### ISSUE TITLE
Auto Scraping Feature

### ISSUE THREAD

**Issue by jpass022** [ionomy/ion#48]
Can an Auto Scraping Feature be implemented for Nodes like we had with the XPY wallet?

**Comment by IngCr3at1on** [comment]
@jpass022 I could probably pick that code from another repo rather easily if it's wanted actually.

**Comment by mitchellcash** [comment]
@IngCr3at1on wait for pent up demand, then a bounty gets announced, then profit ;) haha

**Comment by jpass022** [comment]
Can the scraping be set up so that the wallet has to be unlocked to change the address?

**Comment by IngCr3at1on** [comment]
@jpass022 yes, in fact that was in error that it wasn't done that way on Paycoin. That was actually fixed at one point but I don't remember if it was ever merged. I wound up importing that entire function into another coin per a contract I had and it included the lock requirement.
